cordlib/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/CordaTestBase.kt

```diff
@@ -115,40 +113,50 @@ open class CordaTestBase {
 
     @Before
     fun commonSetup() {
-        TestConfigurationsProvider.provider = object : TestConfigurationsProvider {
-            override fun getConfig(name: String): Configuration? {
-                // Watch carefully for these hard-coded values
-                // Now we assume that issuer(indy trustee) is the first created node from SomeNodes
-                return if (name == "Trustee") {
-                    ConfigurationMap(
-                        mapOf(
-                            "indyuser.walletName" to name,
-                            "indyuser.role" to "trustee",
-                            "indyuser.did" to "V4SGRU86Z58d6TV7PBUe6f",
-                            "indyuser.seed" to "000000000000000000000000Trustee1",
-                            "indyuser.genesisFile" to PoolManager.TEST_GENESIS_FILE_PATH
-                        )
-                    )
-                } else ConfigurationMap(
-                    mapOf(
-                        "indyuser.walletName" to name + random.nextLong().absoluteValue,
-                        "indyuser.genesisFile" to PoolManager.TEST_GENESIS_FILE_PATH
-                    )
-                )
-            }
-        }
-
         net = InternalMockNetwork(
             cordappPackages = listOf("com.luxoft.blockchainlab.corda.hyperledger.indy"),
```

Comment:
how about using Flow package name?
ex. CreateSchemaFlow::class.packageName ?

Reply:
I don't know. Let's discuss.

Reply:
Keep the tests simple, don't overengeneer them.

Reply:
I like it if it works, @seniorjoinu any concern?

Reply:
"it" - I mean Alexey's suggestion

Reply:
The only concern is you have to place here some class that will lay at the top level package of the cordapp. For example if we have structure like `com.luxoft.cordapp`, `com.luxoft.cordapp.flow`, `com.luxoft.cordapp.utils`, `com.luxoft.cordapp.models` and so on, we should have at least 1 class in `com.luxoft.cordapp` that will be placed inside cordappPackages.

I agree with @voddan - there is no actual need in this stuff.